lib/rialto/etl/configs/stanford_organizations_to_sparql_statements.rb

```diff
@@ -11,8 +11,15 @@
 extend Rialto::Etl::Vocabs
 
 def contextualized_org_name(organization)
-  return organization['name'] if organization['parent'].nil? || organization['parent']['name'] == 'Stanford University'
-  "#{organization['name']} (#{organization['parent']['name']})"
+  if ['External Relations',
+      'Administration',
+      "Dean's Office",
+      'Financial Aid',
+      'Financial Aid Office',
+      'Research Centers'].include?(organization['name']) && !organization['parent'].nil?
```

Comment:
How about we stick this array in a constant.  How about `DEPARTMENTS_NEEDING_CONTEXT`?

Reply:
OK. Fixed.